packages/plugin-json-config/index.js

```diff
@@ -0,0 +1,50 @@
+import insertLink from '@octolinker/helper-insert-link';
+import processJSON from '@octolinker/helper-process-json';
+import {
+  jsonRegExValue,
+  jsonRegExArrayValue,
+} from '@octolinker/helper-regex-builder';
+import liveResolverQuery from '@octolinker/resolver-live-query';
+
+function linkDependency(blob, key, value) {
+  let regex;
+
+  if (Number.isNaN(key)) {
```

Comment:
When the value is an array the `key` values are numeric so `jsonRegExValue` doesn't work.